### ISSUE TITLE
Bug: block refs are not displayed correctly in Agenda 3

### ISSUE THREAD

**Issue by abdgad** [haydenull/logseq-plugin-agenda#268]
If you add a block ref to a daily journal, it doesn't appear correctly in Agenda 3.

Example:

![image](https://github.com/haydenull/logseq-plugin-agenda/assets/24955909/471e35bf-f59f-4a96-be6c-52fac762db95)

![image](https://github.com/haydenull/logseq-plugin-agenda/assets/24955909/61d70374-ea6f-431d-acfe-755178912015)

In Agenda 3, it's displayed as "block" instead of "Report a bug":

![image](https://github.com/haydenull/logseq-plugin-agenda/assets/24955909/047a331b-3c16-439a-933a-1838a3ffce6c)

It would be nice if the block ref is displayed correctly.
Also it would be nice if Agenda 3 displays block refs of TODOs without having to prepend TODO to them.
For example:

![image](https://github.com/haydenull/logseq-plugin-agenda/assets/24955909/41905862-6ba6-4a09-b29e-5f4d91a9403c)

This example is not even seen as a task by Agenda 3. It would be nice if Agenda 3 recognizes and treats this block ref as a normal task.

**Comment by haydenull** [comment]
1. Display the detailed content of block ref us in the development plan.

2. The original block is marked with a TODO, which identifies it as a task. I would like to understand why the reference to this block also needs to be recognized as a task, as it may result in duplicate tasks.

**Comment by abdgad** [comment]
>     2. The original block is marked with a TODO, which identifies it as a task. I would like to understand why the reference to this block also needs to be recognized as a task, as it may result in duplicate tasks.

This should only happen in the daily journal page. It's basically for scheduling the original TODO for the date of the daily journal.
Here is an example.
Let's say there a "Projects" page like this:

![image](https://github.com/haydenull/logseq-plugin-agenda/assets/24955909/b1f343a9-73d2-41f3-a5dc-2c9b5f19bd05)

We copy the TODO block ref to the daily journal page:

![image](https://github.com/haydenull/logseq-plugin-agenda/assets/24955909/96ccf8aa-c5c4-41b0-a5f7-a337ac5fefbe)

Now the original TODO should be scheduled for "07-12-2023" and should appear in the Agenda.
So in a nutshell, it's for scheduling the original TODO using only its block ref without creating a new TODO.

**Comment by haydenull** [comment]
‌Agenda can now display the content of block ref correctly.

However, since a block can be referenced by multiple journal pages, the task date still needs to be manually specified.
